spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -80,32 +103,57 @@ definitions:
             type: CarrierPhase
             units: cycles
             desc: Carrier phase observation with typical sign convention.
+        - D:
```

Comment:
@anth-cole explained to me it doesn't really impact RTK since we can't use measured doppler anyway (too inaccurate)